src/schema/v2/partner_category.ts

```diff
@@ -1,36 +1,42 @@
 import _ from "lodash"
 import cached from "./fields/cached"
-import Partners from "./partners"
-import CategoryType from "./input_fields/category_type"
+import PartnerCategoryTypeEnum from "./input_fields/partner_category_type"
 import { SlugAndInternalIDFields } from "./object_identification"
 import {
   GraphQLString,
   GraphQLObjectType,
   GraphQLNonNull,
   GraphQLFieldConfig,
+  GraphQLBoolean,
 } from "graphql"
 import { ResolverContext } from "types/graphql"
 
-const PartnerCategoryType = new GraphQLObjectType<any, ResolverContext>({
+export const PartnerCategoryType = new GraphQLObjectType<any, ResolverContext>({
   name: "PartnerCategory",
-  fields: () => ({
-    ...SlugAndInternalIDFields,
-    cached,
-    category_type: CategoryType,
-    name: {
-      type: GraphQLString,
-    },
-    partners: {
-      type: Partners.type,
-      args: Partners.args,
-      resolve: ({ id }, options, { partnersLoader }) =>
-        partnersLoader(
-          _.defaults(options, {
-            partner_categories: [id],
-          })
-        ),
-    },
-  }),
+  fields: () => {
+    const Partners = require("./partners").default
```

Comment:
Why did this change from a regular `import` to this `require` style instead?

Reply:
I think @alloy updated this because cyclical imports don't play well with debugging in node.

Reply:
Circular require dependencies 😞 Doing it here means it won’t get loaded until the `fields` ‘thunk’ (a lazy function) is evaluated.